{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:35:21"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:35:21"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:35:21"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:37:00"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:37:00"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:37:00"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:37:11"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:37:11"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:37:11"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:37:41"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:37:41"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:37:41"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:38:39"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:38:39"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:38:39"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:39:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:39:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:39:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:41:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:41:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:41:38"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:42:35"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:42:35"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:42:35"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:43:32"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:43:33"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:43:33"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:43:43"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:43:43"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:43:43"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:43:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:43:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:43:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:44:13"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:44:13"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:44:13"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:45:14"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:45:14"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:45:14"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:45:53"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:45:53"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:45:53"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:46:08"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:46:08"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:46:08"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:46:19"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:46:19"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:46:19"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:46:28"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:46:28"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:46:28"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:47:00"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:47:00"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:47:00"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:47:36"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:47:36"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:47:36"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:48:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:48:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:48:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:48:17"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:48:17"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:48:17"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:48:30"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:48:30"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:48:30"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-10 11:48:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-10 11:48:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-10 11:48:55"}
